tests/post/steps/test_example_solution.py

```diff
@@ -0,0 +1,331 @@
+INFRA_TAINT = "node-role.kubernetes.io/infra:NoSchedule-"
+BOOTSTRAP_TAINT = "node-role.kubernetes.io/bootstrap:NoSchedule-"
+SOLUTION_CONFIGMAP = "metalk8s-solutions"
+SOLUTION_NAMESPACE = "metalk8s-solutions"
+
+# }}}
+
+# Fixtures {{{
+
+@pytest.fixture
+def k8s_appsv1_client(k8s_apiclient):
+    return AppsV1Api(api_client=k8s_apiclient)
+
+
+@pytest.fixture(scope='function')
+def context():
+    return {}
+
+
+# }}}
+
+
+# Scenarios {{{
+
+@scenario('../features/example_solution.feature', 'Deploy Solution')
+def test_deploy_solution(host):
+    pass
+
+# }}}
+
+
+# Given {{{
+
+@given(parsers.parse("no Solution '{name}' is imported"))
+def is_absent_solution(host, name, k8s_client):
+    with host.sudo():
+        assert name not in host.mount_point.get_mountpoints(), \
+            'No Solution Configuration file exists at {}'.format(name)
+
+
+@given(parsers.parse("no Solution environment '{environment}' is available"))
+def is_absent_solution_environment(host, environment, k8s_client):
+    solution_environment = get_environment(k8s_client, environment)
+    assert solution_environment is None, \
+        'Found Solution environment {}'.format(environment)
+
+
+@given(parsers.parse("the Solution Configuration file is absent"))
+def is_absent_solution_config(host):
+    with host.sudo():
+        if host.file(SOLUTION_CONFIGURATION_FILE).exists:
+            assert False, \
+                'Solution Config file exists at path {}'.format(
+                    SOLUTION_CONFIGURATION_FILE
+                )
+
+# }}}
+
+
+# When {{{
+
+@when(parsers.parse("we import a Solution archive '{path}'"))
+def import_solution(request, host, path):
+    run_solutions_command(request, host, ["import", "--archive", path])
+
+
+@when(parsers.parse("we activate Solution '{name}' version '{version}'"))
+def activate_example_solution(request, host, name, version):
+    run_solutions_command(
+        request,
+        host,
+        ["activate", "--name", name, "--version", version]
+    )
+
+
+@when(parsers.parse(
+    "we create a solution environment '{environment}'"))
+def create_solution_environment(request, host, environment):
+    run_solutions_command(request, host, ["create-env", "--name", environment])
+
+
+@when("we remove Taints on Bootstrap node before deployment")
+def remove_bootstrap_node_taints(request, host):
+    # Remove Taints on the Bootstrap node before deploying
+    # example-solution-operator else, the operator pods will not startup
+
+    with host.sudo():
+        res = host.run((
+            'kubectl --kubeconfig=/etc/kubernetes/admin.conf '
+            'taint nodes --all {}'
+        ).format(BOOTSTRAP_TAINT))
+        assert res.rc == 0, res.stdout
+
+    with host.sudo():
+        res = host.run((
+            'kubectl --kubeconfig=/etc/kubernetes/admin.conf '
+            'taint nodes --all {}'
+        ).format(INFRA_TAINT))
+        assert res.rc == 0, res.stdout
```

Comment:
Should either be `remove_nodes_taints()` or better: you should only remove taints on bootstrap node here (not using the `--all flag`)